addons/mail/static/src/models/messaging_notification_handler/messaging_notification_handler.js

```diff
@@ -747,15 +750,21 @@ function factory(dependencies) {
                     const channelNameWithIcon = channelIcon + channelName;
                     notificationTitle = _.str.sprintf(
                         this.env._t("%s from %s"),
-                        owl.utils.escape(authorName),
+                        escapedAuthorName,
                         channelNameWithIcon
                     );
                 } else {
-                    notificationTitle = owl.utils.escape(authorName);
+                    notificationTitle = escapedAuthorName;
                 }
             }
-            const notificationContent = message.prettyBody.substr(0, PREVIEW_MSG_MAX_SIZE);
-            this.env.services['bus_service'].sendNotification(notificationTitle, notificationContent);
+            const notificationContent = mailUtils.htmlToTextContentInline(message.prettyBody);
+            this.env.services['bus_service'].sendNotification({
+                headerIconDisable: true,
+                message: notificationContent,
+                title: notificationTitle,
+                titleColor: '#212529',
```

Comment:
same reasoning, I don't like this color is copy/pasted around like it's a random value when we actually have a variable for it